### ISSUE TITLE
Has anybody used this on a Chromebook Pixel 2?

### ISSUE THREAD

**Issue by jayarc** [corna/me_cleaner#140]
I'd like to flash my Pixel 2, i7 Broadwell to neutralize the ME. Looking for others who have tried. I'm wanting to buy the hardware for flashing but first would love to know if hardware flashing is needed for this machine.

**Comment by 65a** [comment]
Yes, hardware flashing is basically required and a normal clip won't work.
